### PR TITLE
Cut the 0.10.7 release

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ola (0.10.7-1) unstable; urgency=low
+
+  * New upstream release
+
+ -- Simon Newton <nomis52@gmail.com>  Tue, 10 Jul 2018 02:32:00 +0000
```
Comment (nomis52) [inline debian/changelog]: Best change this.

Comment (peternewman) [reply debian/changelog]: Same as https://github.com/OpenLightingProject/ola/pull/1361#discussion_r160041808 :stuck_out_tongue:

**Comment by peternewman** [comment]
Also just pausing to see if https://groups.google.com/d/topic/rdm-testing/8DGMrzirlhA/discussion needs fixing before we release.

**Comment by peternewman** [comment]
Yes, it looks like https://github.com/OpenLightingProject/ola/pull/1052 needs backporting.

**Comment by peternewman** [comment]
In which case https://github.com/OpenLightingProject/ola/blob/0.10/debian/tests/control should probably be extended to have an ola-rdm-tests test suite that runs rdm_responder_test.py

**Comment by peternewman** [comment]
Probably ready for take 2, aside from whatever happens with https://groups.google.com/d/topic/rdm-testing/8DGMrzirlhA/discussion .

File: debian/changelog (hunk 1 above)
Comment (nomis52) [reply debian/changelog]: Please change this.

Comment (peternewman) [reply debian/changelog]: Done

**Pull request merged**